.eslintrc.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
         'class-methods-use-this': 'off',
         'jsx-a11y/label-has-associated-control': 'off',
         'import/no-extraneous-dependencies': 'off', // fixme
-        'prefer-destructuring': 'off', // fixme
```

Comment:
Can we limit it to enforcing object destructuring only? That way we can continue using `items[0]` notation.

Reply:
We can do that in `@box/frontend`, which this inherits from.